docs/dev/ways-of-working.md

```diff
@@ -88,4 +88,4 @@ In particular:
   ```
 
 ## Gaffer 2
-During the Gaffer 2 development process there is a **v2-alpha** branch, which acts as the **develop** branch for changes that will only be added to Gaffer 2.
+During the Gaffer 2 development process there was a **v2-alpha** branch, which acted as the **develop** branch for changes staged for Gaffer 2.
```

Comment:
Is it worth mentioning that we should ignore that branch and to use `develop` instead?

Reply:
It already said to use develop further up the page. I've made it clear to now ignore `v2-alpha`.